internal/ocm/ocm.go

```diff
@@ -28,20 +28,8 @@ type ClusterServiceClientSpec interface {
 func getDefaultAdditionalProperities() map[string]string {
 	// additionalProperties should be empty in production, it is configurable for development to pin to specific
 	// provision shards or instruct CS to skip the full provisioning/deprovisioning flow.
-	additionalProperties := map[string]string{
-		// Enable the ARO HCP provisioner during development. For now, if not set a cluster will not progress past the
-		// installing state in CS.
-		"provisioner_hostedcluster_step_enabled": "true",
-		// Enable the provisioning of ACM's ManagedCluster CR associated to the ARO-HCP
-		// cluster during ARO-HCP Cluster provisioning. For now, if not set a cluster will not progress past the
-		// installing state in CS.
-		"provisioner_managedcluster_step_enabled": "true",
-
-		// Enable the provisioning and deprovisioning of ARO-HCP Node Pools. For now, if not set the provisioning
-		// and deprovisioning of day 2 ARO-HCP Node Pools will not be performed on the Management Cluster.
-		"np_provisioner_provision_enabled":   "true",
-		"np_provisioner_deprovision_enabled": "true",
-	}
+	additionalProperties := map[string]string{}
+
```

Comment:
I suggest just deleting this whole function and declaring a `map[string]string{}` where `getDefaultAdditionalProperities` is called.

Reply:
That's fine for me if you want. The reason I kept the function is to have it there in case in the future we want to add default properties, to have it contained in that function.

Let me know if you still prefer what you suggested.

Reply:
Understood.  If there's a need in the future for always-present additional properties we can reinstate that function, but I don't foresee it.  So I'd still prefer it removed, please.

Reply:
👍 I will change it then

Reply:
updated